test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -378,7 +378,7 @@ UCS_TEST_P(test_ucp_wireup_1sided, address) {
         if (sender().worker()->context->tl_rscs[tl].flags & UCP_TL_RSC_FLAG_SOCKADDR) {
             continue;
         }
-        packed_dev_priorities.insert(sender().worker()->ifaces[tl].attr.priority);
+        packed_dev_priorities.insert(ucp_worker_iface_get_attr(sender().worker(), tl)->priority);
```

Comment:
@yosefe BTW, this test fails on master on jazz18

Reply:
so is this an unrelated fix?

Reply:
without this fix issue appeared on test-node3 with MRAIL=2

Reply:
push it into separate PR?

Reply:
no, since you said it fails with MRAIL=2
i guess the comment "this test fails on master on jazz18" actually means "this test fails on master on jazz18 with MRAIL=2"?

Reply:
It falls on jazz18 always, on node 3 with rails>1